d3a_interface/enums.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
+
+
+class BidOfferMatchAlgoEnum(Enum):
+    PAY_AS_BID = 1
+    PAY_AS_CLEAR = 2
+    EXTERNAL = 3
+
+
+class SpotMarketTypeEnum(Enum):
```

Comment:
Just saw this. Do you think you could homogenize this with https://github.com/gridsingularity/d3a-web/blob/41bcce49f00dedf83723ac868f0e393ac141236a/simulation/graphql/graphql_enums.py#L28 ?

Reply:
Done